client/src/Pages/checkoutpage/checkoutpage.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
```

Comment:
this is the checkout page, the user is able to select the quantity of the products, as well completed delete the products from the shopping cart.  Stripe button is also implemented in this page.